ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -410,6 +417,7 @@ function getValues(pendingApproval, t, actions, history) {
         ethErrors.provider.userRejectedRequest().serialize(),
       ),
     networkDisplay: !originIsMetaMask,
+    disableAlerts: !(unnaprovedTxsCount > 0),
```

Comment:
Do we have to assume that the `switch-ethereum-chain` would also want this value as it has an alert too and we don't want to change the behaviour?

Reply:
I think this comment is not valid anymore, because we disable alerts in their own context now.